_rules/document-has-main-landmark-b40fd1.md

```diff
@@ -0,0 +1,162 @@
+---
+id: b40fd1
+name: HTML page has a main landmark
+rule_type: atomic
+description: |
+  This rule checks that each page has an element with a semantic role of `main`
+accessibility_requirements:
```

Comment:
Technique is not named

Reply:
That's on purpose 😉 

The related technique is [ARIA11: Using ARIA landmarks to identify regions of a page](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA11) which is more general than just having a `main` landmark. However, I wanted to keep the rule simple, and the presence of a `main` landmark is enough to identify the main content and thus pass 2.4.1 (it's the same thing as having a link to skip to main content for users who can do landmark navigation).

As a result, the rule does not have a fail-on-fail relationship with ARIA11

There is a note (in the Expectations) in this rule to make that point clear.